deeplearning4j/pom.xml

```diff
@@ -58,6 +57,7 @@
         <module>deeplearning4j-zoo</module>
         <module>deeplearning4j-data</module>
         <module>deeplearning4j-common-tests</module>
+        <module>deeplearning4j-parallelwrapper</module>
```

Comment:
Parallelwrapper is its own top level module now?

Reply:
Yes since scaleout doesn't exist. The directory layout doesn't affect users.